### ISSUE TITLE
Accessibility Issue - data-elementor-device-mode="tablet" in desktop editing mode

### ISSUE THREAD

**Issue by diggeddy** [pojome/elementor#3547]
On smaller/lower resolution screens or when browser zoom is engaged the computed screen resolution can can force data-elementor-device-mode = "tablet". Which locks out the desktop section and column UI. 

This data attribute should be defined by the Elementor controls to overcome this issue. 

This potentially limits accessibility for sight impaired and mobile working on smaller laptops.

**Comment by bainternet** [comment]
@diggeddy 

can you provide steps to replicate?

**Comment by bainternet** [comment]
Closing for lack of activity,

Feel free to reopen if the issue persists.

**Comment by diggeddy** [comment]
See here - issue is relate to users on 13" macbook safari browser:

https://www.facebook.com/groups/Elementors/permalink/1407300972729018/
